wpimath/src/main/java/edu/wpi/first/math/controller/SimpleMotorFeedforward.java

```diff
@@ -187,83 +187,11 @@ public <U extends Unit> Voltage calculate(Measure<? extends PerUnit<U, TimeUnit>
   public <U extends Unit> Voltage calculate(
       Measure<? extends PerUnit<U, TimeUnit>> currentVelocity,
       Measure<? extends PerUnit<U, TimeUnit>> nextVelocity) {
-    // For a simple DC motor with the model
-    //
-    //   dx/dt = −kᵥ/kₐ x + 1/kₐ u - kₛ/kₐ sgn(x),
-    //
-    // where
-    //
-    //   A = −kᵥ/kₐ
-    //   B = 1/kₐ
-    //   c = -kₛ/kₐ sgn(x))
-    //   A_d = eᴬᵀ
-    //   B_d = A⁻¹(eᴬᵀ - I)B
-    //   dx/dt = Ax + Bu + c
-    //
-    // Discretize the affine model.
-    //
-    //   dx/dt = Ax + Bu + c
-    //   dx/dt = Ax + B(u + B⁺c)
-    //   xₖ₊₁ = eᴬᵀxₖ + A⁻¹(eᴬᵀ - I)B(uₖ + B⁺cₖ)
-    //   xₖ₊₁ = A_d xₖ + B_d (uₖ + B⁺cₖ)
-    //   xₖ₊₁ = A_d xₖ + B_d uₖ + B_d B⁺cₖ
-    //
-    // Solve for uₖ.
-    //
-    //   B_d uₖ = xₖ₊₁ − A_d xₖ − B_d B⁺cₖ
-    //   uₖ = B_d⁺(xₖ₊₁ − A_d xₖ − B_d B⁺cₖ)
-    //   uₖ = B_d⁺(xₖ₊₁ − A_d xₖ) − B⁺cₖ
-    //
-    // Substitute in B assuming sgn(x) is a constant for the duration of the step.
-    //
-    //   uₖ = B_d⁺(xₖ₊₁ − A_d xₖ) − kₐ(-(kₛ/kₐ sgn(x)))
-    //   uₖ = B_d⁺(xₖ₊₁ − A_d xₖ) + kₐ(kₛ/kₐ sgn(x))
-    //   uₖ = B_d⁺(xₖ₊₁ − A_d xₖ) + kₛ sgn(x)
     if (ka == 0.0) {
-      // Simplify the model when kₐ = 0.
```

Comment:
The source code should reference where the derivation is for discoverability. Ideally, reference the specific section of the derivation document so the developer doesn't have to go digging through the whole thing.

Reply:
I'm assuming the same for all of the FF's ?

Reply:
Yea.

Reply:
`    //derivation at wpimath/algorithms.md#Simple motor feedforward
    if (ka == 0.0) {`

Something like this?

Reply:
```cpp
// See wpimath/algorithms.md#Simple_motor_feedforward for derivation
```